### ISSUE TITLE
BT Support?

### ISSUE THREAD

**Issue by amouly** [matthewcodes/OBDSim#1]
Any update on Bluetooth support?

**Comment by matthewwilson** [comment]
Sorry @amouly I am not actively developing this project. Feel free to do a pull request and implement BT support.

**Comment by matthewwilson** [comment]
Actually had a look in the source code:

There is a line in main.java

`public static boolean USE_BLUETOOTH = false;`

try changing it to true and running the app, this was initial work for bluetooth and may need some tweeks

**Comment by amouly** [comment]
Yes, I saw some files related to BT, that's why I'm asking. I never implemented a BT server, but I'm going to investigate about that. 

Thank you for your help.

**Comment by matthewwilson** [comment]
No problem, hope you get it working! The library I chose for BT was http://bluecove.org/ if that helps.

**Comment by amouly** [comment]
How I can connect to the OBDServer? I started it on the default port, and I want to connect to it with Netcat. Any idea or recommendation? Thanks.

**Comment by amouly** [comment]
It's working with Netcat now!! Forget it ;).
